feat: next server example (#16)
shuttle-hq/examples: axum/static-next-server/README.md

> This example is a template for an [axum](https://github.com/tokio-rs/axum) server that serves a NextJS single-page application as static assets, deployed with shuttle. For more information on how to achieve this, check out this article by one of our contributors:  https://joshmo.hashnode.dev/deploying-a-nextjs-front-end-with-a-rust-api-in-one-go```

Reviewer: ```suggestion
# Serving a NextJS project with Axum

This example is a template for an [axum](https://github.com/tokio-rs/axum) server that serves a NextJS single-page application as static assets, deployed with shuttle. For more information on how to achieve this, check out this article by one of our contributors:  https://joshmo.hashnode.dev/deploying-a-nextjs-front-end-with-a-rust-api-in-one-go```
```

# Serving a NextJS project with Axum

> This example shows how to serve a NextJS project using [axum](https://github.com/tokio-rs/axum) and shuttle.

Reviewer: I would say this is essentially the same as our static files axum example, and not really an example of serving a nextjs project using axum. I know this is supposed to serve as a template for the node cli (and thus the extra bits in the other static files examples shouldn't be included), but it may be confusing for users seeing this in the examples repo. Maybe we can leave instructions for how to extract a next.js app to the static folder, or just link to Josh's article?

> # Serving a NextJS project with Axum
> 
> This example shows how to serve a NextJS project using [axum](https://github.com/tokio-rs/axum) and shuttle.

Reviewer: ```suggestion
This example is a template for an [axum](https://github.com/tokio-rs/axum) server that serves a NextJS single-page application as static assets. For more information on how to achieve this, check out this article by one of our contributors:  https://joshmo.hashnode.dev/deploying-a-nextjs-front-end-with-a-rust-api-in-one-go```

> This example is a template for an [axum](https://github.com/tokio-rs/axum) server that serves a NextJS single-page application as static assets, deployed with shuttle. For more information on how to achieve this, check out this article by one of our contributors:  https://joshmo.hashnode.dev/deploying-a-nextjs-front-end-with-a-rust-api-in-one-go```

Reviewer: ```suggestion
This example is a template for an [axum](https://github.com/tokio-rs/axum) server that serves a NextJS single-page application as static assets, deployed with shuttle. For more information on how to achieve this, check out this article by one of our contributors:  https://joshmo.hashnode.dev/deploying-a-nextjs-front-end-with-a-rust-api-in-one-go
```
